aopy/analysis.py

```diff
@@ -1543,3 +1543,61 @@ def interp_multichannel(x):
     x[nan_idx] = np.interp(idx,xp,fp)
 
     return x
+
+
+'''
+Behavioral metrics 
+'''
+
+def compute_path_length_per_trial(trajectory):
```

Comment:
why is it per trial? doesn't this just take a single trajectory

Reply:
Each trajectory is a cursor trajectory per trial. This function calculates path length per trial instead of path length for all trials in a session. Hence, the name says _per_trial

Reply:
i would call this one "calc_path_length" and the one that does all trials in a session "calc_path_length_per_trial". to me it makes more sense to have the "_per_trial" suffix on a function that cares about trials. this one you just give it a path and it computes the length, it doesn't need to know anything about trial structure.

Reply:
I think _per_trial makes it explicit, and the function explanation says it is for a single trajectory. Other function names like time_to_target() or get_trial_segments() without _per_trial suffix are all used for batch processing. If that makes sense. I want to leave it as it is. If this causes further confusion, we can change it in a future patch.

Reply:
It could be '_per_trajectory' insted of 'per_trial' perhaps. But either way i think a suffix if important.

Reply:
haha. I will update the code to reflect this.